config/services.yml

```diff
@@ -129,6 +133,7 @@ services:
       $productPageChoices: '@=service("prestashop.module.link_block.choice_provider.product_page").getChoices()'
       $staticPageChoices: '@=service("prestashop.module.link_block.choice_provider.static_page").getChoices()'
       $categoryChoices: '@=service("prestashop.module.link_block.choice_provider.category").getChoices()'
+      $multiStoreFeature: '@prestashop.adapter.feature.multistore'
```

Comment:
Maybe I could do 

`isMultiStoreEnabled` and `'@=service("prestashop.adapter.feature.multistore").isUsed()'`

🤔

Reply:
Well, indeed why fetch the big object if you only need one boolean from it 😄

Reply:
agreed